h2o-docs/src/product/explainability.rst

```diff
@@ -0,0 +1,223 @@
+
+Notes
+~~~~~
+
+Our roadmap for improving the explainability interface is [here](https://0xdata.atlassian.net/jira/software/c/projects/PUBDEV/issues/PUBDEV-7806?filter=allissues).
+
+We are looking for feedback from users.  If you have any feature requests, please file a 
+
+
+
+Code Examples
+-------------
+
+See below for code examples in R and Python.  We are working on some longer explainability tutorials that we will link here soon.
+
+
+Explain models
+~~~~~~~~~~~~~~
+
+Here’s an example showing basic usage of the ``h2o.explain()`` function in *R* and the ``explain()`` method in *Python*.  Keep in mind that this code should be run in an environment that can support plots.  We recommend Jypyter notebooks in Python and RStudio IDE in R (either in the console or in R Markdown file or notebook).
+
+
+.. tabs::
+   .. code-tab:: r R
+
+        library(h2o)
+        h2o.init()
+
+        # Import a sample binary outcome train/test set into H2O
+        train <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_train_10k.csv")
+        test <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_test_5k.csv")
+
+        # Identify predictors and response
+        y <- "response"
+        x <- setdiff(names(train), y)
+
+        # For binary classification, response should be a factor
+        train[, y] <- as.factor(train[, y])
+        test[, y] <- as.factor(test[, y])
+
+        # Run AutoML for 20 base models
+        aml <- h2o.automl(x = x, y = y, 
+                          training_frame = train,
+                          max_models = 10,
+                          seed = 1)
+
+        # Explain leader model & compare with all AutoML models                  
+        exa <- h2o.explain(aml, test)
+        exa
+
+        # Explain a single H2O model (e.g. leader model from AutoML)
+        exm <- h2o.explain(aml@leader, test)
+        exm
+
+
+
+   .. code-tab:: python
+
+        import h2o
+        from h2o.automl import H2OAutoML
+        from h2o.explain import explain, explain_row
+        from IPython.core.display import display
+
+        h2o.init()
+
+        # Import a sample binary outcome train/test set into H2O
+        train = h2o.import_file("https://s3.amazonaws.com/erin-data/higgs/higgs_train_10k.csv")
+        test = h2o.import_file("https://s3.amazonaws.com/erin-data/higgs/higgs_test_5k.csv")
+
+        # Identify predictors and response
+        x = train.columns
+        y = "response"
+        x.remove(y)
+
+        # For binary classification, response should be a factor
+        train[y] = train[y].asfactor()
+        test[y] = test[y].asfactor()
+        
+        # Run AutoML for 20 base models
+        aml = H2OAutoML(max_models=10, seed=1)
+        aml.train(x=x, y=y, training_frame=train)
+
+        # Explain leader model & compare with all AutoML models 
+        exa = aml.explain(test)
+        display(exa)
```

Comment:
The `display(exa)` will display the explanation for the second time. The `aml.explain()` will display it since some matplotlib backends don't make it easy to show closed figure and not closing it would make Jupyter automatically showing it.

So if you don't provide `explain` function with `render=False`, it will render it when it runs. 

We could either add a comment saying that `display` will show it for second time or remove the `display`.

The other option could be `aml.explain(test, render=False)` but I would discourage putting it to the example since it makes it hard to show the figures in CLI and could confuse the users.

Reply:
@tomasfryda Thanks for the reminder.. ill update to be clear about this.

Reply:
fixed